### PR TITLE
Added top margin to the index header logo

### PR DESCRIPTION
**Pull request by umarsheikh13**
Salaam. The lack of top margin on the index header logo was flaring up my OCD (sorry) so I've made the necessary changes to fix this. It is a design change so if you like it the way it is that's cool, have a look at the image below.

Will be looking at the other issues and picking up what I can, I have experience in Node.js, React/Native, Sass etc.
index-header-margin-top

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32819

File: src/components/IndexHeader/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default class IndexHeader extends Component {
         <div className="container">
           <div className="row">
             <div className="col-md-10 col-md-offset-1 text-center">
-              <Link to="/" data-metrics-event-name="IndexHeader:Link:Index">
+              <Link to="/" className="link" data-metrics-event-name="IndexHeader:Link:Index">
```
Comment (sabeurthabti) [inline src/components/IndexHeader/index.js]: minor suggestion: we use css-modules via webpack, this allows us to require style styles, and it also means less globals in our css. 

```js

const styles = require('./style.scss')]

<Link to="/" className={styles.link} data-metrics-event-name="IndexHeader:Link:Index">
```

If you have time, this is a preferred approach. 
Example: https://github.com/quran/quran.com-frontend/blob/master/src/components/Ayah/index.js#L13

docs:
https://github.com/css-modules/css-modules

Comment (mmahalwy) [inline src/components/IndexHeader/index.js]: +1 on that

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32822

**Comment by umarsheikh13** [comment]
Thanks for getting back so quick. I've moved the styles into a separate file like you mentioned @sabeurthabti.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32823

**Comment by sabeurthabti** [comment]
@umarsheikh13 nice one Umar!

**Pull request merged**